### PR TITLE
Make debug_tools configurable

### PR DESCRIPTION
**Pull request by hxietkg**
What this PR does / why we need it:
debug_tools is always true, and not configurable, we need an option to disable it.

Which issue(s) this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged): Fixes #

**Additional context**
Add any other context for the reviewers

### CODE REVIEW

File: images/capi/packer/ova/packer-windows.json [hunk 1]
```diff
@@ -264,6 +265,7 @@
     "windows_service_manager": null,
     "windows_updates_categories": null,
     "windows_updates_kbs": null,
-    "wins_url": "https://github.com/rancher/wins/releases/download/v{{user `wins_version`}}/wins.exe"
+    "wins_url": "https://github.com/rancher/wins/releases/download/v{{user `wins_version`}}/wins.exe",
+    "debug_tools": "true"
```
Comment (jsturtevant) [inline images/capi/packer/ova/packer-windows.json:269]: This just needs to be sorted alphabetically.  We should add it to the https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/packer/config/windows/common.json file to keep as default for all providers. With the new field being passed through the Azure configs didn't get the debug files on the CI builds.

**Review by jsturtevant — CHANGES_REQUESTED** [review]
We should add the `debug_tools=true` it to the https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/packer/config/windows/common.json file to keep as default for all providers. With the new field being passed through the Azure configs didn't get the debug files on the CI builds.

File: images/capi/packer/ova/packer-windows.json (hunk 1 above)
Comment (hxietkg) [reply images/capi/packer/ova/packer-windows.json]: done

**Pull request merged**